data_generators/user_generators.py

```diff
@@ -18,6 +18,8 @@ class RandomMentor(Printable):
 
     def __init__(self):
         self.profile_id = generate_uuid(16)
+        self.created_at = "2018-06-12T09:55:22"
+        self.updated_at = "2018-06-12T09:55:22"
```

Comment:
please alter created_at and updated_at variables with the following code: `datetime.now().strftime("%Y-%m-%dT%H:%M:%S")`.

Reply:
We'd like to do this to keep it consistant with the changes made in our `feautre/mentee-generators-update` branch/PR

Reply:
above code snippet is insufficient per Robert. Please use `datetime.datetime.now().isoformat()` instead. We will make the same change in mentee generator.